### PR TITLE
Highlight placeholders in Scenario Outline #60

### PR DESCRIPTION
**Pull request by rscrimojr**

### CODE REVIEW

File: radish/extensions/console_writer.py [hunk 1]
```diff
@@ -45,6 +46,8 @@ def __init__(self):
         self.last_precondition = None
         self.last_background = None
 
+        self._placeholder_regex = re.compile(r'(<\w+>)', flags=re.UNICODE)
```
Comment (timofurrer) [inline radish/extensions/console_writer.py]: You can actually use more characters then only the ones included in `\w`, like a hyphen.

File: radish/extensions/console_writer.py [hunk 2]
```diff
@@ -225,7 +228,16 @@ def console_writer_after_each_step(self, step):
 
         color_func = self.get_color_func(step.state)
         line_jump_seq = self.get_line_jump_seq() * (((len(step.raw_text) + 3) if step.text else 1) + (len(step.table) if step.table else 0))
-        output = u"{0}        {1}{2}".format(line_jump_seq, self.get_id_sentence_prefix(step, colorful.bold_cyan), color_func(step.sentence))
+        output = u'{0}        '.format(line_jump_seq)
+
+        if isinstance(step.parent, ScenarioOutline):
+            # Highlight Scenario Outline placeholders e.g. '<method>'
+            output += (u''.join([str(colorful.bold_white(item) if (self._placeholder_regex.search(item)
```
Comment (timofurrer) [inline radish/extensions/console_writer.py]: I think it's too dominant in bold. I suggest to only use `colorful.white`

Comment (timofurrer) [inline radish/extensions/console_writer.py]: You don't need the `[]` for the list comprehension. `str.join()` can work with generators directly.

File: radish/extensions/console_writer.py [hunk 3]
```diff
@@ -225,7 +228,16 @@ def console_writer_after_each_step(self, step):
...
+                                 and item.strip('<>') in step.parent.examples_header)
```
Comment (timofurrer) [inline radish/extensions/console_writer.py]: I'm not quite sure but would it be possible to just use `re.sub` and substitute with a colored string?

File: radish/extensions/console_writer.py (hunk 1 above)
Comment (rscrimojr) [inline radish/extensions/console_writer.py]: Okay. I will update the regex

File: radish/extensions/console_writer.py (hunk 2 above)
Comment (rscrimojr) [inline radish/extensions/console_writer.py]: no problem. Will update

Comment (rscrimojr) [inline radish/extensions/console_writer.py]: Okay. I will change this

File: radish/extensions/console_writer.py (hunk 3 above)
Comment (rscrimojr) [inline radish/extensions/console_writer.py]: re.sub does not work unless you add the default color codes after the substitutions to change the color back to the standard color. Which is essentially what this solution accomplishes  e.g. `When I <method> the <fruit>.` . "When I", "the" and the "." need to be the standard color.

**Comment by rscrimojr** [comment]
@timofurrer Is their something I need to do with the requested changes to let you know I completed them or is the comments and commit enough?  Thanks.

**Comment by timofurrer** [comment]
Looks good! Thanks for your contribution! 

I'll release it soon!

**Comment by rscrimojr** [comment]
You're welcome.  I am glad I could help out.

**Pull request merged**